recipes/pulse/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pulse" %}
+{% set version = "2019.0" %}
+{% set sha256 = "db2a150e2f6476657642bf7094f6a3912b5841775edbd96c3adbcfa3456c41e6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/finsberg/pulse/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  noarch: python
+
+requirements:
+  build:
+    - h5py>=2.8.0
+    - fenics 2018
+    - scipy
+    - matplotlib
+  run:
+    - h5py>=2.8.0
+    - fenics 2018
+    - scipy
+    - matplotlib
+test:
+  imports:
+    - pulse
+
+about:
+  home: https://finsberg.github.io/pulse/html/index.html
+  license: LGPL-3.0
+  license_family: LGPL
+  license_file: LICENSE
+  summary: 'pulse is a python library based on FEniCS that aims to solve problems in continuum mechanics'
+
+  description: |
+    pulse is a python library based on FEniCS that aims to solve
+    problems in continuum mechanics, in particular cardiac mechanics.
+  doc_url: https://finsberg.github.io/pulse/html/index.html
+  dev_url: https://github.com/finsberg/pulse
```

Please, add at least yourself as the maintainer of this recipe
```suggestion
  dev_url: https://github.com/finsberg/pulse
extra:
  recipe-maintainers:
    - finsberg
```